snet_cli/commands.py

```diff
@@ -510,8 +510,8 @@ def call(self):
         spec_hash = ClientCommand(
             config=self.config,
             args=self.args,
-            out_f=None,
-            err_f=None,
+            out_f=self.err_f,
```

Comment:
Why self.err_f is used for  out_f instead of self.out_f?

Reply:
There are other places where out_f and err_f are set to None.
For example ContractCommand: https://github.com/singnet/snet-cli/blob/master/snet_cli/commands.py#L302
Should these places updated as well?

Reply:
My guess is that the previous dev team use `self.out_f` (stdout) only for output transaction data.
For "basic" logging they use the `self.err_f` (stderr).
Like they did [here](https://github.com/singnet/snet-cli/blob/841b6d3bc3a2fa9c5b0856ae2ecb4a732b4ee949/snet_cli/commands.py#L589-L590)

Reply:
> There are other places where out_f and err_f are set to None.
> For example ContractCommand: https://github.com/singnet/snet-cli/blob/master/snet_cli/commands.py#L302
> Should these places updated as well?

I think so...cause I prefer more logs then less.
The downside is that will be a bunch of logs from "intermediary" transactions.
Maybe we can use this in a `--verbose` mode.

Reply:
It sounds good. It would be better to move it into a separate issue.